po/pt_BR.po

```diff
@@ -0,0 +1,119 @@
+"POT-Creation-Date: 2024-02-29 23:09+0300\n"
+"PO-Revision-Date: 2024-03-05 20:03-0300\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: pt_BR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.4.2\n"
+
+#: data/io.github.zefr0x.hashes.desktop.in:5
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:6 hashes/ui/main_window.py:29
+msgid "Hashes"
+msgstr "Hashes"
+
+#: data/io.github.zefr0x.hashes.desktop.in:6
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:7
+msgid "Identify hashing algorithms"
+msgstr "Identificar algorítimos de hashing"
+
+#: data/io.github.zefr0x.hashes.desktop.in:7
+msgid "Hash Identifier"
+msgstr "Identificador de Hash"
+
+#: data/io.github.zefr0x.hashes.desktop.in:13
+msgid "hash;hashcat;john;detect;identify"
+msgstr "hash;hashcat;john;detectar;identificar"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:9
+msgid ""
+"This utility lets you conveniently find out what hashing algorithm has been "
+"used on a particular string."
+msgstr ""
+"Essa ferramenta permite que você descubra convenientemente qual algoritmo "
+"de hash foi usado em uma string específica."
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:10
+msgid "Features:"
+msgstr "Recursos:"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:12
+msgid "📺 Popularity Ratings - Most popular hashes showed first."
+msgstr ""
+"📺 Classificações de popularidade - Os hashes mais populares são mostrados "
+"primeiro."
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:13
+msgid ""
+"👵 Updated! - A new hash algorithm database is used in the identification "
+"process."
+msgstr ""
+"👵 Atualizado! - Um novo banco de dados de algoritmos de hash é usado no "
+"processo de identificação."
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:17
+msgid "zefr0x"
+msgstr "zefr0x"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:26
+msgid "hash"
+msgstr "hash"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:27
+msgid "hashcat"
+msgstr "hashcat"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:28
+msgid "john"
+msgstr "john"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:29
+msgid "detect"
+msgstr "detectar"
+
+#: data/io.github.zefr0x.hashes.metainfo.xml.in:30
+msgid "identify"
+msgstr "identificar"
+
+#: hashes/ui/main_window.py:48
+msgid "Enter a hash text"
+msgstr "Entre um texto de hash"
+
+#: hashes/ui/main_window.py:63
+msgid "Waiting for input…"
+msgstr "Esperando por input…"
+
+#: hashes/ui/main_window.py:65
+msgid "Can't find an algorithm that matches your input!"
+msgstr "Não foi possível encontrar um algorítimo que combine com seu input!"
+
+#: hashes/ui/main_window.py:154
+msgid "<b>Extended!</b>"
+msgstr "<b>Extendido!</b>"
+
+#: hashes/ui/main_window.py:165
+msgid "Hashcat"
+msgstr "Hashcat"
+
+#: hashes/ui/main_window.py:179
+msgid "John"
+msgstr "John"
+
+#: hashes/ui/main_window.py:208
+msgid "translator-credits"
+msgstr "créditos-de-tradutor"
```

Comment:
This should contain your name and optional email or URL

Take a look at other files for an example